Automattic/WP-Job-Manager
# Register and use original tinymce link and not use wplink.

Fixes #920 

#### Changes proposed in this Pull Request:

* Instead of modify the search query in wplink (tinymce), simply register and use "link" tinymce plugin.

#### Testing instructions:

* Check this video: https://drive.google.com/file/d/0B9TdkMXV_Q7GS1R2aHNtUVUydlE/view?usp=sharing


#### Proposed changelog entry for your changes:

* Use "link" tinymce plugin.
